rr_common/cfg/lin-tracking-reconf.cfg

```
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
+PACKAGE = "rr_common"
+
+from dynamic_reconfigure.parameter_generator_catkin import *
+
+gen = ParameterGenerator()
+
+# Linear Tracking Filter Varibles (arbitrary max and min values)
+gen.add("val_max", double_t, 0, "linear model: maximum value", 0, 0.0, 30.0)
+gen.add("val_min", double_t, 0, "linear model: minimum value", 0, -5.0, 0.0)
+gen.add("rate_max", double_t, 0, "linear model: maximum rate", 0, 0.0, 50.0)
+gen.add("rate_min", double_t, 0, "linear model: minimum rate", 0, -50.0, 0.0)
+
+
+exit(gen.generate(PACKAGE, "rr_common", "LinearTracking"))
```

```suggestion
exit(gen.generate(PACKAGE, "rr_common", "lin-tracking-reconf"))
```
If the third parameter is different from the cfg filename, excluding the extension, then the libraries will be generated in every build forcing a recompilation of the nodes that use them.